### PR TITLE
home work 1, complete reg user scenario

### PR DESCRIPTION
**Pull request by ihappynavi**

### CODE REVIEW

File: tests/register_test.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 Feature('register');
 
 const NEW_USER = {
-    firstName: "Oleg",
-    lastName: "9",
-    email: Date.now() + '@test.com',
+    firstName: "Ivan",
+    lastName: "37",
+    emailField: 'ihappynavi' + Date.now() + '@test.com',
```
Comment (adudyak) [inline tests/register_test.js]: `email`
remove "Field" at the end, it is not required for object option

File: tests/register_test.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+    telephoneField: '380732761986',
+    passwordField: 'PAss21!!',
+    passwordConfirmField: 'PAss21!!',
```
Comment (adudyak) [inline tests/register_test.js]: one password is enough, remove extra one

File: tests/register_test.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 Feature('register');
 
 const NEW_USER = {
-    firstName: "Oleg",
-    lastName: "9",
-    email: Date.now() + '@test.com',
+    firstName: "Ivan",
+    lastName: "37",
+    emailField: 'ihappynavi' + Date.now() + '@test.com',
+    telephoneField: '380732761986',
```
Comment (adudyak) [inline tests/register_test.js]: remove "Field" at the end, it is not required for object option

File: tests/register_test.js [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
+    passwordField: 'PAss21!!',
```
Comment (adudyak) [inline tests/register_test.js]: remove "Field" at the end, it is not required for object option

File: tests/register_test.js [hunk 5]
```diff
@@ -12,11 +15,8 @@ Scenario('register new user', ({ I, basePage, registerPage }) => {
     basePage.clickMyRegisterLink();
     registerPage.verifyRegisterPage();
     registerPage.fillNewUserForm(NEW_USER);
+    registerPage.clickPolicyAgree();
+    registerPage.submitRegisterForm();
     pause();
```
Comment (adudyak) [inline tests/register_test.js]: remove `pause()`, it is for debugging, should not exist in repo

**Review by adudyak — APPROVED** [review]
Looks ok. Use Alt + Shift + F to fix spacing in files.

File: tests/register_test.js (hunk 1 above)
Comment (ihappynavi) [reply tests/register_test.js]: done

File: tests/register_test.js (hunk 3 above)
Comment (ihappynavi) [reply tests/register_test.js]: done

**Pull request closed without merging**